src/components/logic/calculator.js

```diff
@@ -4,7 +4,7 @@ function isNumber(item) {
   return !!item.match(/[0-9]+/);
 }
 
-export default function calculate(obj, buttonName) {
+const calculate = (obj, buttonName) => {
```

Comment:
- [ ] This is not the change requested. You should refactor the `src/components/calculator` component, not this logic file. Kindly look carefully at the file names you are changing. You should change the component file, not logic